locales/fr_FR/pcsx2_Main.po

```diff
@@ -2143,7 +2135,7 @@ msgstr "Utiliser les paramètres par défaut"
 
 #: pcsx2/gui/Panels/DirPickerPanel.cpp:174
 msgid "Open in Explorer"
-msgstr "Afficher dans l'explorateur"
+msgstr "Afficher dans l'explorateur Windoxs"
```

Comment:
Windows :)

Reply:
Whoops, haven't seen that one coming. Fixing this.